third-party/README

```diff
@@ -11,9 +11,9 @@ chpl-venv/
   Summary: Directory where several python packages are downloaded and
            installed. Used by chpldoc and the testing system, and to generate
            the man page and standard module documentation.
-           See chpldoc-requirements.txt, test-requirements.txt, and
-           virtualenv.txt files for a complete list of packages that are
-           installed.
+           See chpldoc-requirements.txt, test-requirements.txt,
+	   scspell-requirements.txt and virtualenv.txt files for a
+	   complete list of packages that are installed.
 
```

Comment:
Oops, I didn't notice this from the github web diff, but I messed up tabs vs. spaces here. I'll fix that too.